android/app/src/main/java/it/teamdigitale/app/italiaapp/MainActivity.java

```diff
@@ -40,8 +40,18 @@ protected void onCreate(Bundle savedInstanceState) {
             alertDialog.setCancelable(false);
             alertDialog.show();
         } else {
-            SplashScreen.show(this, R.style.SplashScreenTheme);
-            super.onCreate(savedInstanceState);
+            if (getResources().getBoolean(R.bool.isTablet)) {
```

Comment:
Since we have to show messages on AlertsDialog (not cancelable) could you add a generic method that accepts title and message? The handler must be always the activity finishing

Then I suggest you could check if the device is a tablet before we check if it is an emulator or the device is rooted.

Reply:
Done